src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -147,6 +151,8 @@ class CrudField {
     window.crud = {
         ...window.crud,
 
+        initializeFields: true,
+
```

Comment:
This could also be something like;

```suggestion
        config: {
            fields: {
                initializeByDefault: true,
                triggerChangeOnInput: true,
            }
        }

```

```
crud.config.fields.initializeByDefault = false;
```